courses/BIOL_103.json

```diff
@@ -13,7 +13,7 @@
             "Eng Sci": 0, 
             "Math": 0
         }, 
-        "career": "Distance Studies", 
+        "career": "Undergraduate", 
```

Comment:
In a future PR, we should move career onto sections instead of courses IMO, so that we can have BIOL 103 for both Distance Studies and Undergraduate.